src/services/production.ts

```diff
@@ -149,73 +104,28 @@ export const services: ServiceConfigurator<SupportedTokens> = ($) => [
     ],
   }],
 
-  // App
-  [$.app, {
-    service: useApp,
-    arguments: [
-      $.store,
-      $.router,
-      $.components,
-    ],
-  }],
-  // Routes
-  [$.routes, {
-    service: routes,
-    arguments: [
-      $.routesLabel,
-    ],
-  }],
-
-  [$.meshRoutes, {
-    service: meshRoutes,
-    arguments: [
-      $.serviceRoutes,
-      $.gatewayRoutes,
-      $.dataplaneRoutes,
-      $.policyRoutes,
-    ],
-    labels: [
-      $.routesLabel,
-    ],
-  }],
-
-  [$.dataplaneRoutes, {
-    service: dataplaneRoutes,
-  }],
-  [$.gatewayRoutes, {
-    service: gatewayRoutes,
-  }],
-  [$.serviceRoutes, {
-    service: serviceRoutes,
-  }],
-  [$.policyRoutes, {
-    service: policyRoutes,
-  }],
-
-  [$.zoneRoutes, {
-    service: zoneRoutes,
-    arguments: [
-      constant(
-        [...(import.meta.env.VITE_ZONE_CREATION_FLOW === 'enabled' ? zoneActionRoutes() : [])],
-        {
-          description: 'kuma.zone.action.routes',
-        },
-      ),
-    ],
-    labels: [
-      $.routesLabel,
-    ],
-  }],
-
-  // Modules
-  ...application($),
-  ...mainOverviewModule($),
-  ...zonesModule($),
-  ...meshes($),
-  ...servicesModule($),
-  ...dataplanes($),
-  ...gateways($),
-  ...policies($),
+  // application
+  ...vue($),
+  ...application({
+    ...$,
+    routes: $.routesLabel,
+  }),
+  //
+
+  // service-mesh
+  ...mainOverview({
+    ...$,
+    routes: $.routesLabel,
+  }),
+  ...zones({
+    ...$,
+    routes: $.routesLabel,
+  }),
+  ...meshes({
+    ...$,
+    routes: $.routesLabel,
+  }),
```

Comment:
I think this could end up being a `serviceMesh` plugin/module/micro-frontend.

That would mean that in each app, you would only add the single `serviceMesh` plugin to get all of this, then you can pick and choose what extras you want ontop of that.